shopify_app.gemspec

```diff
@@ -8,9 +8,7 @@ Gem::Specification.new do |s|
   s.author      = "Shopify"
   s.summary     = %q{This gem is used to get quickly started with the Shopify API}
 
-  s.add_dependency('rails', '>= 4.2.6', '< 5.0')
```

Comment:
@swalkinshaw to be honest I don't know for sure what are overriding rules for gemspec Gemfile combo (if there are any), and to speak freely I was happy to not know such thing, and to not force others to know such things, while reading files. Having same thing configured in 2 places == need for knowing overriding rules. Again -- feel free to change as it suits you